pkg/virtualKubelet/reflection/manager/manager_test.go

```diff
@@ -54,7 +55,9 @@ var _ = Describe("Manager tests", func() {
 	})
 	AfterEach(func() { cancel() })
 
-	JustBeforeEach(func() { mgr = New(localClient, remoteClient, localLiqoClient, remoteLiqoClient, 1*time.Hour) })
+	JustBeforeEach(func() {
+		mgr = New(localClient, remoteClient, localLiqoClient, remoteLiqoClient, 1*time.Hour, record.NewBroadcaster())
```

Comment:
Could add an Expect assertion in lines 70 and 122 to assert the broadcaster is correctly stored and propagated.